src/utils/features/copyArticle.ts

```
@@ -0,0 +1,261 @@
+        case 'code':
+          // Inline code
+          if (!el.parentElement || el.parentElement.tagName.toLowerCase() !== 'pre') {
+            parts.push('`' + (el.textContent || '') + '`');
+          } else {
+            // Block code - handled by pre
+            parts.push(el.textContent || '');
+          }
+          break;
+        case 'pre':
+          // Code block
+          const codeEl = el.querySelector('code');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)